pom.xml

```diff
@@ -2282,7 +2282,7 @@
     <properties>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.25.342</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.25.346-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
`SNAPSHOT` is this intentional?

Reply:
Need to bump to the correct version after merging the dependent PRs

Reply:
TODO: Version bump

Reply:
Done